cluster/gce/list-resources.sh

```diff
@@ -75,10 +75,10 @@ echo "Provider: ${KUBERNETES_PROVIDER:-}"
 # provided.
 gcloud-compute-list instance-templates --regexp="${INSTANCE_PREFIX}.*"
```

Comment:
I didn't switch this one as `--regexp` isn't deprecated for that command yet, happy to preemptively remove it though. I'll check for any others that can be switched to filter as well.